website_wow/static/tests/js/website_wow.editor.tour.js

```diff
@@ -0,0 +1,108 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.tour_editor', function (require) {
+    'use strict';
+
+    var tour = require("web_tour.tour");
+    var base = require("web_editor.base");
+    var snippet_editor = require('web_editor.snippet.editor');
+
+    require('website_wow.editor');
+    require('website_wow.view');
+
+    tour.register("website_wow_editor", {
+        url: '/page/contactus?debug=assets&enable_editor=1',
+        test: true,
+        wait_for: base.ready(),
+    }, [{
+        content: 'Create an animable element.',
+        trigger: 'div[name=Cover]',
+        run: function () {
+            var $snippet = $('section.s_text_block_image_fw');
+            var $div = $('<div class="oe_structure o_editable o_dirty note-air-editor note-editable" data-oe-id="239" data-oe-model="ir.ui.view" data-oe-field="arch" data-oe-xpath="/t[1]/t[1]/div[1]/div[3]" data-note-id="1" contenteditable="true"></div>');
+            $div.append($snippet);
+            $('main').append($div);
+            $snippet.addClass('o_dirty');
+            // Save the element that we created
+            $('button[data-action=save]').click();
+            // But our element causes some weird error, so discard (it's still saved somehow...)
+            $('button[data-action=cancel]').click();
+            // Click OK
+            $('span:contains("Ok")').parent().click();
+        }
+    }, {
+        content: 'Go back to edit',
+        trigger: 'a[data-action=edit]',
+        run: function () {
+            $('a[data-action=edit]').click();
+        }
+    }, {
+        content: 'Click an animable element.',
+        trigger: 'main h2:contains("Headline")',
+        run: function () {
```

Comment:
In my by-hand tests, the customize menu appears automatically after drag and dropping, so possibly you can skip this step. Otherwise, maybe better `run: "click"`